### ISSUE TITLE
[TW-1583] Invalid ID displayed in "task all" for deleted or completed tasks

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1609]
_Ulf Eliasson on 2015-03-26T20:25:29Z says:_ 

 After deleting a task, the first execution of reports such as `task all` displays the wrong ID.

Subsequent executions display the correct ID.

{code}
$  task add Task 1 && task add Task 2 && task add Task 3
Created task 1.
Created task 2.
Created task 3.

$  task all                                             

ID St UUID     Age D Description
 1 P  d2d1865e 2s    Task 1
 2 P  140ecdf5 2s    Task 2     
 3 P  77346eab 2s    Task 3

3 tasks

$  task 1 done
Completed task 1 'Task 1'.
Completed 1 task.

$  task 3 delete
Permanently delete task 3 'Task 3'? (yes/no) yes
Deleting task 3 'Task 3'.
Deleted 1 task.

$  task all

ID St UUID     Age Done D Description
 1 P  6e77fd65 9s         Task 2
 1 C  fe9eb48d 9s  7s     Task 1     
 3 D  91783532 9s  3s     Task 3

3 tasks

$  task all

ID St UUID     Age Done D Description
 1 P  6e77fd65 11s        Task 2
*  C  fe9eb48d 11s 9s     Task 1     
*  D  91783532 11s 5s     Task 3

3 tasks
{code}

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-03-26T20:25:29Z
Modified: 2015-03-28T09:53:03Z
```

**Comment by taskwarrior** [comment]
_Wim Schuermann on 2015-03-28T09:53:03Z says:_ 

 Fixed in [9e6c6ecb93b65f5b13aba76f760e825fa9256f93|https://git.tasktools.org/projects/TM/repos/task/commits/9e6c6ecb93b65f5b13aba76f760e825fa9256f93]

Thanks for reporting this!
